remind.py

```diff
@@ -395,16 +395,19 @@ def to_remind(self, vevent, label=None, priority=None, tags=None):
         elif isinstance(trigdates, str):
             remind.append(trigdates)
 
+        def abbr_tag(tag):
```

Comment:
My inclination would be to leave it as a small anonymous function, but here you go.